lib/container.js

```diff
@@ -283,8 +284,29 @@ function loadGherkinSteps(paths) {
   global.After = fn => event.dispatcher.on(event.test.finished, fn);
   global.Fail = fn => event.dispatcher.on(event.test.failed, fn);
 
-  for (const path of paths) {
-    loadSupportObject(path, `Step Definition from ${path}`);
+  // If gherkin.steps includes only one *.js, then this will iterate through that folder and send all step def js files to loadSupportObject
+  // This is done so that we need not enter all Step Definition files under config.gherkin.steps
+  // This works only for one level folder, not for subfolders
+  if (paths.length === 1 && paths[0] === './step_definitions/*.js') {
```

Comment:
Sorry, I don't like this `if` block and especially that you hardcode `'./step_definitions/*.js'`.
Can you do it in a different way?

Reply:
I have made loop decision based on isArray. Thus have removed the if block.